src/metaprob_viz/viz.clj

```diff
@@ -217,15 +218,24 @@
 (defn set-port! [p]
   (reset! port p))
 
-(defn start-server! [ & [p]]
-  (set-port! (or p 8081))
-  (reset! server (httpkit/run-server
-                  (-> #'routes
-                      (wrap-ws ws-handler)
-                      wrap-viz)
-                  {:port @port})))
+(defn set-ip! [i]
+  (reset! ip i))
```

Comment:
This is interesting. Do we anticipate people preferring to use `set-ip!` followed by `start-server!` rather than just providing the IP to `start-server!` directly? 🤔

Reply:
Not sure! _Probably_ not? I suspect people will mostly use `start-server!`. This could probably be removed.